app/scripts/models/auth_brokers/fx-desktop-v1.js

```diff
@@ -0,0 +1,40 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/**
+ * A broker that knows how to communicate with Firefox when used for Sync.
+ */
+
+'use strict';
+
+define([
+  'models/auth_brokers/fx-desktop',
+  'lib/url',
+  'lib/channels/fx-desktop'
+], function (FxDesktopBroker, Url, FxDesktopChannel) {
+
+  var FxDesktopV1AuthenticationBroker = FxDesktopBroker.extend({
```

Comment:
I'm not convinced of the V1/V2 naming - V2 appends a `fxaccounts:` prefix to all the messages, so that seems like a sufficiently large change to warrant a version change. The other difference between V1/V2 is the channel types change. Perhaps this can all be done in the fx-desktop broker and which channel is created is decided by a configuration parameter.